distribution/pom.xml

```diff
@@ -34,7 +34,7 @@
         <module>docs</module>
         <module>ddf-catalog</module>
         <module>ddf</module>
-        <module>docker</module>
+        <!--module>docker</module-->
```

Comment:
we should be able to remove this change.  This was to get around a temporary build issue.